staging/src/k8s.io/library-go/scheduling/corev1helpers/README.md

```diff
@@ -0,0 +1,6 @@
+# sig-scheduling/corev1helpers
+
+Primarily to be imported by the scheduling framework to allow
```

Comment:
This should be expressed as a criteria, rather than motivation.

Also, can it just be a package godoc?

Reply:
> This should be expressed as a criteria

Descheduler/Cluster capacity are not part of the framework. Yet, they are another suitable candidates for importing the helpers. Although, this is a step toward to have the framework be importable as a go module, it's still important to allow other components to import the code. Otherwise, we might just as well duplicate the code and put it under the framework.

> Also, can it just be a package godoc?

How many people read godoc vs. readme? I'd like the text to be visible so people are aware. Although, how many people read anything but code comments.

Reply:
> This should be expressed as a criteria, rather than motivation.

So I would say something like:

```
Functions in this package should satisfy one of the following:

- Be used by more than one core component (kube-scheduler, kubelet, kube-apiserver, etc.)
- Be used by a core component and another kubernetes project (cluster-autoscaler, descheduler)

And be a scheduling feature.

See initial motivation: ....
```

> How many people read godoc vs. readme? I'd like the text to be visible so people are aware. Although, how many people read anything but code comments.

A code comment is more visible when you are reviewing code changes. After all, the audience of this is developers and reviewers.

Reply:
Moved under `doc.go` with your suggested comment.